vignettes/Making_maps_with_cancensus.Rmd

```diff
@@ -0,0 +1,132 @@
+---
+title: "Making maps with cancensus"
+author: "Dmitry Shkolnik"
+date: "11/20/2017"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(cancensus)
+library(sf)
+
+# Preload census data for Toronto, CT level, median household income v_CA16_2397
```

Comment:
Why preload the data here? If you want to speed up development while working on the vignette, you're better off passing the `cache = TRUE` parameter to the knitr block below. Otherwise you'll need to keep the two blocks in sync manually, anyway. Or is there another reason?

Reply:
Even with the change to the quiet parameter, the various calls produce messaging when executed. I've played around with the `results` and `messages` parameters and I find the output not that pleasing. I wanted to keep the output in this case to just the maps, so I decided to evaluate the census call in a hidden chunk and then mock the calls in unevaluated chunks below.